app/js/csvAddShow.js

```diff
@@ -66,7 +65,22 @@ document.getElementById('chooseCSV').addEventListener('click', () => {
                 const newColumn = document.createElement('td');
                 newColumn.appendChild(associateAudioButton);
 
-                const timeFrameLengthInput = document.createElement('input');
+                const timeFrameLengthInput = document.createElement('select');
+                const option25  = document.createElement('option');
+                const option50  = document.createElement('option');
+                const option100 = document.createElement('option');
+
+                option25.text   = 25;
```

Comment:
Shouldn't text have quotes around them and value be a number? Also change the assignment order to be consistent to text then value.